ssz/sedes/integer.py

```diff
@@ -0,0 +1,67 @@
+from ssz.exceptions import (
+    DeserializationError,
+    SerializationError,
+)
+
+
+class Integer:
+    """
+    A sedes for integers (Handles both int<N> and uint<N>).
+    """
+    # This variable captures 'N' in 'int<N>' or 'uint<N>'
+    num_bytes = 0
+    allows_signed_ints = True
```

Comment:
1. Suggestion: rename to `is_signed`
2. Set to unsigned by default.

Reply:
@hwwhww I have removed the support for `signed integers`, as suggested by jannikluhn. So this won't be a matter anyway. But I could put back the support for signed integers if you want. What's your call?

Reply:
I think it's okay to remove signed integers. 👍